tools/codegen/dest/native_functions.py

```diff
@@ -53,3 +53,49 @@ def compute_native_function_declaration(
     else:
         x = gen_unstructured(g, backend_index)
         return [] if x is None else [x]
+
+
+@with_native_function_and_index
+def gen_unstructured_lazy_definition(f: NativeFunction, backend_index: BackendIndex) -> Optional[str]:
+    sig = kernel_signature(f, backend_index)
+    metadata = backend_index.get_kernel(f)
+    
+    # Lazy IR stuff
+    all_types, value_types, scalar_types = process_ir_types(f.func)
+
+    lazy_tensors = "\n    ".join([f"LazyTensor l_{t.name} = bridge::GetLtcTensor({t.name});" for t in value_types])
+    node_ctor_inputs = ", ".join([f"l_{t.name}" for t in value_types] +
```

Comment:
It doesn't seem there is any particular reason to give locals for these, since they immediately get passed into MakeNode and are never subsequently used. Saves you from having to explicitly `std::move` them in too.

Reply:
yea, i agree.  i guess i was just wary of making one line of code that was too hard to read.